topics/contributing/tutorials/create-new-tutorial/tutorial.md

```diff
@@ -256,7 +256,7 @@ For the next times, you can make it quicker.
 >             --workflow_id "ID of the workflow on the Galaxy instance" \
 >             --zenodo_link "URL to the Zenodo record"
 >      ```
->    - option 2: from a local workflow file (`.ga`)
+>    - option 2: from a local workflow file (`.ga`) (use only if your workflow is composed of tools from the main ToolShed)
```

Comment:
That is not really true I guess. The crucial point is that the tools in your workflow need to be installed in the Galaxy that you use. Where the tools coming from should not matter, but they need to be installed.

Reply:
Hmmm, really strange effectively, but using .ga local workflow file seems to have this issue if I well understood https://github.com/galaxyproject/training-material/issues/1518 B! ;) comment